modules/build/src/test/scala/scala/build/tests/BuildTests.scala

```diff
@@ -260,13 +260,7 @@ abstract class BuildTests(server: Boolean) extends munit.FunSuite {
           |""".stripMargin
     )
     testInputs.withBuild(defaultOptions, buildThreads, bloopConfigOpt) { (_, _, maybeBuild) =>
-      maybeBuild.orThrow.assertGeneratedEquals(
-        "simple.class",
-        "simple_sc.class",
-        "simple$.class",
-        "simple_sc$.class"
-      )
-      maybeBuild.orThrow.assertNoDiagnostics
+      expect(maybeBuild.isLeft)
```

Comment:
Add some content, that from now `ivy` and `dep` are not supported.  It should be added for every test, when you change `assert`, that now you expected fail with `ivy` and `dep` dependency.

Reply:
You mean test names? If so, I changed them.

Reply:
yeah, it better now.